### DISCUSSION TITLE
Tasmota on LSC Ceiling light does not work

### DISCUSSION THREAD

**Discussion by henkiejan1** [arendst/Tasmota#12844] [Q&A] [answered]
Hi Everyone,

A few years i have now a LSC ceiling light from action. I have removed the Realtek chip and soldered a ESP12-E on it. It works so far so good with ESP Easy.

But because i prefer now to use Tasmota, i flashed it with with Tasmota. It's up and running now but i didn't get the light working again like befoere with  ESP-Easy.

There is a [template here](https://templates.blakadder.com/lsc_smart_connect_ceiling_light.html): but that is different light i think.  In the template the speak about RGB and other GPIO pins. The light i have supports only warm to cold white and is dimmable. What i also know is that it uses GPIO 5 and GPIO13. I have put both in PWM mode but doesn't work...

Can someone help me?

**Comment by Jason2866** [comment, marked as answer]
The LSC lights i know (dont have any myself) are using a light controller chip. So any configuration with PWM mode will not work. There is a detailed issue for https://github.com/arendst/Tasmota/issues/6495

**Reply by henkiejan1** [reply]
I have it working now. It works for sure with PWM. There is now a newer version i see. With RGB. The reason it not work at the beginning was a bad connection to the LED board.

I have only one question left: I use PWM 1 for GPIO5 and PWM 2 for GPIO13. And it works all but the only problem is that when tick on yellow the light is white and when i tick on white it's yellow.

Any idea how i could fix this detail?

Edit: if i am right is the template:
`{"NAME":"LSC Light","GPIO":[0,0,0,0,0,416,0,0,0,417,0,0,0,0],"FLAG":0,"BASE":18}`

**Reply by Jason2866** [reply]
Have you tried to swap pwm1 and pwm2 ?
